content/en/docs/best-practices/channel-naming.md

```diff
@@ -175,9 +202,62 @@ candidate/fast/stable to denote the maturity level of the operator.  Using all
 
 Channel promotion is the notion of moving an operator from one channel to 
 another.  For example, consider the case where you have an operator version 
-1.0.1 which is found in a candidate channel, then you might decide to offer 
+`1.0.1` which is found in a candidate channel, then you might decide to offer 
 support for that version and want to move it to a stable channel.  
 
 Today, channel promotion is achieved by creating a new operator version 
-(1.0.2) that is labeled with the channel(s) you want to promote to (as well 
+(`1.0.2`) that is labeled with the channel(s) you want to promote to (as well 
 as any channels you want to keep it in).
+
+# UPGRADES 
+
+OLM provides a variety of ways to specify updates between operator versions. Before we continue with the following 
+recommendations, make sure you understand the options by checking [Creating an update graph with OLM][upgrade-graph].
+
+## Recommended upgrade path
+
+Within a channel each patch release should be directly upgradable to the [HEAD of channel](/docs/glossary/#channel-head). 
+Use skips or skipRange to provide this behaviour. (i.e. if you followed the above `Example 3` that means 
```

Comment:
We should probably link to the `skips` and `skipsRange` sections of the doc site from here

Reply:
Usually, I would agree with. But then it can bring confusion Why?
skip/skipRange/replaces configured via CSV still supported and valid indeed with FBC format. 
However, our latest version of this doc ONLY explains how to do it with the new FBC options and we remove completed this explanation about how to do it using the CSV.